Makefile

```diff
@@ -36,11 +37,11 @@ endif
 
 vet :
 	@echo "== vet"
-	@go vet $(pkgs)
+	@go vet $(prod_pkgs)
```

Comment:
You can run vet on them yourself to see. But basically it fails because I execute cancelFunc() in the tear down method. But vet wants it to be in the same function. There isn't any way to avoid that.